pylint/test/functional/deprecated_methods_py36.py

```diff
@@ -0,0 +1,40 @@
+""" Functional tests for method deprecation. """
```

Comment:
Ideally, this file should contain only the code for the deprecations happening in the same Python version, which means, in this case you should test only for the deprecation of FileLoader.load_module.